Week2/homework/map-filter.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  const oddNumbers = numbers.filter(number => number % 2 !== 0).map(number => number * 2);
+  return oddNumbers;
```

Comment:
Introducing the variable `oddNumbers` here does not add value. You can simply return the result straightaway, e.g.:

```js
return numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```